frontend/src/features/admin-form/settings/components/EmailFormSection.tsx

```diff
@@ -48,22 +79,42 @@ export const EmailFormSection = ({
     [initialEmailSet, mutateFormEmails],
   )
 
-  useEffect(() => reset({ emails: initialEmails }), [initialEmails, reset])
+  useEffect(() => reset({ emails: settings.emails }), [settings.emails, reset])
+
+  const emailModeDescription = `Add at least **2 recipients** to prevent loss of response.`
+  const storageModeDescription = `FormSG securely stores responses in an encrypted format and does not retain any associated emails.`
 
   return (
     <>
-      <MRFAdvertisingInfobox />
+      <EmailNotificationsHeader
+        isFormPublic={isFormPublic}
+        isPaymentsEnabled={isPaymentsEnabled}
+        isFormResponseModeEmail={
+          settings.responseMode === FormResponseMode.Email
+        }
+      />
       <FormProvider {...formMethods}>
-        <FormControl isInvalid={!isEmpty(errors)}>
+        <FormControl
+          isInvalid={!isEmpty(errors)}
+          isDisabled={isFormPublic || isPaymentsEnabled}
+        >
           <FormLabel
-            isRequired
+            isRequired={settings.responseMode === FormResponseMode.Email}
             useMarkdownForDescription
-            description={`Add at least **2 recipients** to prevent loss of response. Learn more on [how to guard against email bounces](${GUIDE_PREVENT_EMAIL_BOUNCE}).`}
+            description={
+              (settings.responseMode === FormResponseMode.Email
```

Comment:
/nit I think we can also make this to a constant that's evaluated outside of this return

Reply:
Fixed!